src/game/client/ui.cpp

```diff
@@ -99,6 +107,8 @@ void CUI::ClipEnable(const CUIRect *r)
 void CUI::ClipDisable()
 {
 	Graphics()->ClipDisable();
+	m_ClipRect = {};
```

Comment:
I get this error: 

```warning: extended initializer lists only available with -std=c++11 or -std=gnu++11```

Seems like initalizer lists can only be used at initialization